sass/exercises/functions/src/sass/app.css

```diff
@@ -0,0 +1,220 @@
+.btn {
```

Comment:
We should not mix style file extensions, refactor all `.css` into `.scss` extension